lib/storage.js

```diff
@@ -32,6 +32,10 @@ export default class Storage {
   getUniversal (key) {
     // Local state
     let value
+    if (!process.static) {
```

Comment:
With new `target` mode, we should also check that we are not in development also since `process.static` will also be set in dev:

```suggestion
    if (process.dev || !process.static) {
```

Reply:
Isn't the purpose of `process.static` to enforce the same behavior between dev and generated version? This way still we may have different behavior between dev/static...

Reply:
I didn't know of this change. Thanks for the heads up @Atinux